packages/vue-components/src/directives/Closeable.js

```diff
@@ -2,46 +2,53 @@ module.exports = {
-      $showLabel.show();
-      $closeButton.hide();
-      $content.get()[0].style.display = 'none';
+      showLabel.style.display = '';
+      closeButton.style.display = 'none';
+      content.style.display = 'none';
     }
 
     function onShow() {
       el.dataset.isShown = 'true';
-      $showLabel.hide();
-      $content.get()[0].style.display = '';
+      showLabel.style.display = 'none';
+      content.style.display = '';
     }
 
     function onMouseOver() {
       if (el.dataset.isShown === 'false') {
         return;
       }
-      $closeButton.show();
+      closeButton.style.display = '';
     }
 
     function onMouseOut() {
       if (el.dataset.isShown === 'false') {
         return;
       }
-      $closeButton.hide();
+      closeButton.style.display = 'none';
     }
 
     el.dataset.isShown = 'true';
     el.style.position = 'relative';
     const message = el.getAttribute('alt') || 'Expand Content';
-    const $content = jQuery(`<div class="content"></div>`);
-    jQuery(el).contents().appendTo($content);
-    jQuery(el).empty();
-    jQuery(el).append($content);
-    jQuery(el).attr('class', `${el.className} closeable-wrapper`);
-    const $closeButton = jQuery('<span class="closeable-button label label-default hidden-print" style="display: none; position: absolute; top: 0; left: 0; cursor: pointer;background: #d9534f;"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></span>');
-    jQuery(el).append($closeButton);
-    const $showLabel = jQuery(`<a class="closeable-show hidden-print" style="display: none; cursor: pointer;text-decoration: underline">${message}</a>`);
-    jQuery(el).append($showLabel);
-    $closeButton.click(onClose);
-    $showLabel.click(onShow);
-    jQuery(el).on('mouseover', onMouseOver);
-    jQuery(el).on('mouseout', onMouseOut);
+    const content = document.createElement('div');
+    content.classList.add('content');
+    Array.from(el.children).forEach(child => content.append(child));
+    el.replaceChildren();
+    el.append(content);
+    el.setAttribute('class', `${el.className} closeable-wrapper`);
+    const closeButton = document.createElement('span');
+    closeButton.classList.add('closeable-button', 'label', 'label-default', 'hidden-print');
+    closeButton.style.cssText += 'display: none; position: absolute; top: 0; left: 0; cursor: pointer;background: #d9534f';
+    closeButton.innerHTML = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span>';
+    el.append(closeButton);
+    const showLabel = document.createElement('a');
+    showLabel.classList.add('closeable-show', 'hidden-print');
+    showLabel.style.cssText += 'display: none; cursor: pointer;text-decoration: underline';
+    showLabel.innerHTML = message;
+    el.append(showLabel);
+    closeButton.addEventListener('click', onClose);
+    showLabel.addEventListener('click', onShow);
+    el.addEventListener('mouseover', onMouseOver);
+    el.addEventListener('mouseout', onMouseOut);
```

Comment:
Old 33 -> new 33-34
- Create a div with content as a class 

Old 34 -> new 35
- All the children in el was added $content

Old 35 -> new 36
- children is replaced with empty array which is same as empty

Old 36 -> new 37
- $content appended to el

Old 37 -> new 38 
- Add class 

Old 38 -> new 39-42
- Create closeButton with corresponding css and classes

Old 39 -> new 43
- Append close button to el 

Old 40 -> new 44-47
- Create $showLabel 

Old 41 -> new 48 
- Append showLabel 

Old 42 -> new 49 
- Add event listener for click for closeButton

Old 43 -> new 50 
- Add event listener for click for showLabel 

Old 44-45 -> new 51 - 52
- Add eventlistener for mouseover and mouseout

Reply:
can we use some newline to logically split the code into blocks

Reply:
Refactored in [941e9d7](https://github.com/MarkBind/markbind/pull/2130/commits/941e9d78e0a21a27aecc1a002927e3c9b2d26fb6)

Ignore the [WIP]